test/instagram.processor.test.ts

```
@@ -0,0 +1,89 @@
+      },
+      'DXeh-kYiIge'
+    );
+    expect(sub).toBeTruthy();
+    expect(sub!.type).toBe('substatus');
+    expect(sub!.parent_id).toBe('DXeh-kYiIge');
+    expect(sub!.provider).toBe('instagram');
+    expect(sub!.replying_to?.status).toBe('DXeh-kYiIge');
+  });
+
+  it('maps comment edges', () => {
+    const edges = [{ node: { pk: '1', text: 'a', created_at: 1, user: { pk: '2', username: 'u' } } }];
```

ESLint

[error] Replace `{·node:·{·pk:·'1',·text:·'a',·created_at:·1,·user:·{·pk:·'2',·username:·'u'·}·}·}` with `⏎······{·node:·{·pk:·'1',·text:·'a',·created_at:·1,·user:·{·pk:·'2',·username:·'u'·}·}·}⏎····`

(prettier/prettier)